portfolio/src/main/java/com/google/sps/servlets/MarkerServlet.java

```diff
@@ -9,6 +9,7 @@
 import com.google.gson.Gson;
 import com.google.sps.data.Location;
 import com.google.sps.data.Marker;
+import com.google.sps.data.Waypoint;
```

Comment:
you don't need this import. nothing in this file uses it

Reply:
Oh you're right, I forgot to delete it from there, thank you